src/ReactiveDomain.Policy.Tests/PolicyMapTest.cs

```diff
@@ -36,41 +56,51 @@ public class PolicyMapTest
                 );
 
             //building an optional additional policy
-            var backupRole = new Role(backupRoleName, new Permission(typeof(ExportItem)));
+            backupRole = new Role(backupRoleName, new Permission(typeof(ExportItem)));
             var backupAgentPolicy = new Policy(backupPolicyName, backupRole);
 
             //creating the base policy map
-            var policymap = new PolicyMap(policy);
-            policymap.AddAdditionalPolicy(backupAgentPolicy);
-
+            policyMap = new PolicyMap(policy);
+            policyMap.AddAdditionalPolicy(backupAgentPolicy);
 
-            
+            //create usesrs
             // In usage user details and roles will be recieved from the authenication and permissions system
-            var systemUser = new UserDetails
+            systemUser = new UserDetails
             {
                 UserId = Guid.NewGuid(),
                 UserName = "Bob",
                 PolicyName = defaultPolicyName,
                 RoleNames = new string[] { userRoleName, customRoleName }
-            };          
+            };
 
-            var adminUser = new UserDetails
+            adminUser = new UserDetails
             {
                 UserId = Guid.NewGuid(),
                 UserName = "Mike",
                 PolicyName = defaultPolicyName,
                 RoleNames = new string[] { adminRoleName }
             };
-            var backupAgent = new UserDetails
+            backupAgent = new UserDetails
             {
                 UserId = Guid.NewGuid(),
                 UserName = "backupAgent",
                 PolicyName = backupPolicyName,
                 RoleNames = new string[] { backupRoleName }
             };
-
+            dispatcher = new PolicyDispatcher(new Dispatcher(nameof(PolicyMapTest)), () => CurrentPolicy);
+            dispatcher.Subscribe<AddItem>(this);
+            dispatcher.Subscribe<DisplayItem>(this);
+            dispatcher.Subscribe<DeleteItem>(this);
+            dispatcher.Subscribe<ExportItem>(this);
+            dispatcher.Subscribe<OtherCmd>(this);
+            dispatcher.Subscribe<OtherMsg>(this);
+        }
+        [Fact]
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
Any particular reason for this? I understand wanting to allow method names with underscores, but this results in some variables with confusing names that make the code harder to read.

Reply:
Our test method naming convention triggers the warning. Disabling for the file overcorrects though.
I've added a suppression file that targets just the method names